clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/health/KubernetesHealthIndicator.java

```diff
@@ -45,11 +59,14 @@ public KubernetesHealthIndicator(
 
   @Override
   protected Optional<String> accountHealth(KubernetesNamedAccountCredentials accountCredentials) {
-    try {
-      accountCredentials.getCredentials().getDeclaredNamespaces();
-      return Optional.empty();
-    } catch (RuntimeException e) {
-      return Optional.of(e.getMessage());
+    if (kubernetesConfigurationProperties.isVerifyAccountHealth()) {
+      try {
+        accountCredentials.getCredentials().getDeclaredNamespaces();
+        return Optional.empty();
```

Comment:
I suppose it minimizes diffs to keep this, but I'd probably remove it to reduce the duplication with the new `return Optional.empty();` below.